app/services/return-requirements/submit-setup.service.js

```diff
@@ -26,6 +27,7 @@ const SetupValidator = require('../../validators/return-requirements/setup.valid
  */
 async function go (sessionId, payload) {
```

Comment:
Again sorry I know you didn't write the JSDoc comment for this file but boy scout rules and all that... Can you check line 25 and remove the extra 'the' . 'If no errors a **_the_** url' -> 'If no errors a url'